idaes/models/unit_models/__init__.py

```diff
@@ -40,6 +40,7 @@
 )
 from .shell_and_tube_1d import ShellAndTube1D, ShellAndTubeInitializer
 from .skeleton_model import SkeletonUnitModel, SkeletonUnitModelData
+from .stream_scaler import StreamScaler
```

Comment:
WE generally include both the main class and the data class here.

Reply:
Okay thanks for the feedback